### ISSUE TITLE
[export] some logos get exported to zero sized pngs

### ISSUE THREAD

**Issue by Stunkymonkey** [Komet/MediaElch#909]
**Describe the bug**
when exporting some pngs are empty. on the commandline following is printed:
```
...
libpng error: profile 'HD 709-A': 805D79A3h: length does not match profile
libpng error: profile 'HD 709-A': 805D79A3h: length does not match profile
[MediaElch] WARNING:  libpng warning: iCCP: known incorrect sRGB profile
libpng error: profile 'sRGB IEC61966-2.1': 40500441h: length does not match profile
[MediaElch] WARNING:  libpng warning: iCCP: known incorrect sRGB profile
libpng error: profile 'sRGB IEC61966-2.1': 909291A2h: length does not match profile
libpng error: profile 'HD 709-A': 306B70A3h: length does not match profile
libpng error: profile 'Generic RGB Profile': 90B568A3h: length does not match profile
[MediaElch] WARNING:  libpng warning: iCCP: known incorrect sRGB profile
libpng error: profile 'sRGB IEC61966-2.1': 909291A2h: length does not match profile
libpng error: profile 'Generic RGB Profile': 90B568A3h: length does not match profile
[MediaElch] WARNING:  libpng warning: iCCP: known incorrect sRGB profile
libpng error: profile 'sRGB IEC61966-2.1': 909291A2h: length does not match profile
[MediaElch] INFO:     [Export] Finished successfully
...
```

**To Reproduce**

1. Go to Export
1. Click on Export, but make sure the theme exports the movie-logo as well.
1. Open folder and inspect files

**Expected behavior**
I expect all pngs to be exported with valid image-data.
Maybe the images are corrupted. I do not know how to deal with it if this is the case. 

**MediaElch Version:**
 - [ ] 2.6.5-dev (nightly)
 - [x] 2.6.2 stable

**Operating System:**
 - [ ] Windows
 - [ ] macOS
 - [x] Linux (distribution: arch)

**Comment by Stunkymonkey** [comment]
all warnings are thrown at:
[SimpleEngine.cpp#L506](https://github.com/Komet/MediaElch/blob/master/src/export/SimpleEngine.cpp#L506)
errors all at:
[SimpleEngine.cpp#L508](https://github.com/Komet/MediaElch/blob/master/src/export/SimpleEngine.cpp#L508)

**Comment by bugwelle** [comment]
Thanks. We should check whether `size` is valid...

https://github.com/Komet/MediaElch/blob/af8201b1bc3c02e11e9e16c593b1dfcc82cdbb35/src/export/SimpleEngine.cpp#L526-L527

**Comment by Stunkymonkey** [comment]
same applies to 2.6.4

**Comment by bugwelle** [comment]
This may even be a bug in Qt. I found a bug report that reads exactly like this issue. What Qt version do you have installed?

**Comment by Stunkymonkey** [comment]
```
$ pacman -Q qt5-base
qt5-base 5.14.1-3
```

**Comment by Stunkymonkey** [comment]
can you please link the Qt-bug?
this would help for keeping track of it.

**Comment by bugwelle** [comment]
Oh, sorry. Should have linked it right away... Can't find it anymore... 🙅

**Comment by bugwelle** [comment]
Found it again: Likely https://bugreports.qt.io/browse/QTBUG-81671

Seems to be fixed in Qt 5.14.2 and later. As you use 5.14.1, I think you ran into exactly this issue.

MediaElch also no longer uses "format" and "quality" parameters, so another potential mismatch was removed.

I can't reproduce this issue anymore, so I'm closing it. If it appears again, please let me know.

PS: Sorry about the delay. I only now get to go through all open bugs.
